style.css

```diff
@@ -0,0 +1,263 @@
+@import url('https://fonts.googleapis.com/css2?family=Redressed&family=Roboto:wght@300&family=Satisfy&family=Shizuru&display=swap');
+
+body{
+font-family: 'Roboto', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color: #111827;
+}
+
+* {
+
+--primary-color: #ecfeff ;
+--secondary-color: #FA0874;
+}
+
+.container{
+    color: var(--primary-color);
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-template-rows: 200px 200px 200px; 
+ 
+    grid-template-areas: 
+    "nav nav nav"
+    ".. main .. "
+    ".. main  .. "
+    ".. main .. "
+    ".. main .. "
+    " footer footer footer";
+}
+
+nav{
+    display: flex;
+    gap: 20px;
+    width: 100vw;
+    flex-direction: column;
+    grid-area: nav;
+    
+
+}
+
+nav h2{
+    position: relative;
+    top: 35px;
+    font-size: 2rem;
+    line-height: 2.5rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Redressed', cursive;
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+color: var(--secondary-color);
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto; 
+    display: flex;
+  gap: 15px;
+  position: relative;
+  bottom: 35px;
+  padding-inline-end: 0.7rem;
+ 
+}
+
+li{
+ 
+    list-style: none;
+    padding: 0.5rem;
+    font-weight: bold;
+   
+}
+li a{
+    text-decoration: none;
+    color: var(--primary-color);
+    
+}
+
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 3.5rem;
+    text-align: center;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    grid-area: main;
+    font-size: 1.75rem;
+   
+}
+
+footer{
+    grid-area: footer;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem;
+    border-top: 1px solid var(--primary-color);
+    margin-top: 3rem;
+      
+
+}
+
+footer ul{
+ padding-inline-start: 0.5rem;
+    display: flex;
+  gap: 20px;
+
+
+} 
+ 
+
+h3{
+    font-size: 1.5rem;
+    line-height: 2rem;
+    letter-spacing: 0.1rem;
+ }
+
+ .impact{
+     font-style: italic;
+     color: var(--secondary-color);
+     
+ }
+
+ main p{
+     letter-spacing: 0.1rem;
+ }
+
+ .get-started{
+     padding: 0.8rem 1rem;
+     color: var(--secondary-color);
+     border-radius: 1rem;
+     background-color: var(--primary-color);
+     border: 0;
+    font-weight: bold;
+     font-size: 1.2rem;
+     margin: 0.5rem;
+     text-decoration: none;
+ letter-spacing: 0.1rem;
+ }
+
+ 
+
+ .get-started:hover{
+     color: var(--primary-color);
+     background-color: var(--secondary-color);
+ }
+
+
+
+ @media (max-width: 1536px){
+
+    html{
+        font-size: 100%;
+    }
+    
+    main{
+        font-size: 2rem;
+    }
+    
+ }
+
+
+ 
+ @media (max-width: 1280px){
+
+    html{
+        font-size: 92%;

```

Comment:
we don't use % for font-size, its a wrong convention, use `rem` or `em` or `px` instead.